include/ginkgo/core/solver/bicg.hpp

```diff
@@ -108,6 +108,8 @@ class Bicg : public EnableLinOp<Bicg<ValueType>>, public Preconditionable {
     };
     GKO_ENABLE_LIN_OP_FACTORY(Bicg, parameters, Factory);
     GKO_ENABLE_BUILD_METHOD(Factory);
+    GKO_ENABLE_SET_GET_PARAMETERS(
+        stop_criterion_factory, std::shared_ptr<const stop::CriterionFactory>);
```

Comment:
Should we keep the same name in the factory?

Reply:
The set and get functions generated are for the member variables, so I would have to pass in the names and types of the member variables.